library/src/main/kotlin/build/buf/connect/protocols/ConnectConstants.kt

```diff
@@ -28,3 +28,10 @@ const val GRPC_STATUS_DETAILS_TRAILERS = "grpc-status-details-bin"
 const val GRPC_STATUS_TRAILER = "grpc-status"
 const val GRPC_TE_HEADER = "te"
 const val GRPC_USER_AGENT = "x-user-agent"
+
+class ConnectConstants private constructor() {
+    companion object {
```

Comment:
Wouldn't this be simpler to write `object ConnectConstraints {` instead of an empty `class` with `private constructor`?

Reply:
Yup much cleaner - I've written more Java than Kotlin so appreciate you looking out for this sort of thing.